internal/reserved_keywords.go

```diff
@@ -0,0 +1,99 @@
+package internal
+
+var reservedKeywords = map[string]struct{}{
```

Comment:
Where can we have this list?

Reply:
Please add a link for the list.

Reply:
Thank you for your comment.
I fixed.
Please check it.